runtime/runtime.go

```diff
@@ -213,12 +213,16 @@ func PrependEnvsPrefix(cmd string, env map[string]string, soft bool) (string, er
 
 // BootCmdForScript returns boot command that is to be used
 // to run config set with name bootName.
-func BootCmdForScript(bootName string) string {
-	if bootName == "" {
+func BootCmdForScript(bootNames []string) string {
+	if len(bootNames) == 0 {
 		return ""
 	}
+	bootCmd := ""
+	for _, bootName := range bootNames {
+		bootCmd += fmt.Sprintf(";runscript /run/%s", strings.TrimSpace(bootName))
```

Comment:
Replace with `runscript /run/%s;` instead. Hopefully, you won't even need to trim the `;` at the end.

Reply:
You're correct, OSv is not bothered by trailing `;`. Thanks, nice one!